signing_clients/apps.py

```diff
@@ -243,7 +243,6 @@ def __str__(self):
         ).decode('utf-8')
 
 
-@python_2_unicode_compatible
```

Comment:
This decorator is already applied in `Manifest` and is duplicated here, leading to recursive errors.